graviti/manager/branch.py

```diff
@@ -81,18 +81,25 @@ def create(self, name: str, revision: Optional[str] = None) -> Branch:
 
         Raises:
             ForbiddenError: When create branches on the default branch without commit history.
+            TypeError: When the input name or revision is not of string type.
 
         Returns:
             The :class:`.Branch` instance with the given name.
 
         """
+        if not isinstance(name, str):
```

Comment:
shall we handle this task by a framework? like a decorator?

Reply:
here is some references:
- https://github.com/h2oai/typesentry
- https://github.com/agronholm/typeguard
- https://github.com/RussBaz/enforce